.buildkite/pipeline.yml

```diff
@@ -1,6 +1,5 @@
 steps:
   - label: Build and Upload to TestFlight
-    branches: "main"
```

Comment:
I removed this to test my changes, but I think we can remove it anyway given the setting is in the pipeline configuration in our Infrastructure as Code repo